examples/pass-parameters-to-tests/index.js

```diff
@@ -0,0 +1,33 @@
+import JSONconfig from './config/config.json';
+import JSconfig from './config/config';
+import { ClientFunction } from 'testcafe';
+
+const getDocumentUri = ClientFunction(() => document.documentURI);
+
+fixture `Pass parameters to tests`
+
+process.env.Url = './src/env.html';
+
+test('Get URL from an environment variable', async t => {
+    console.log(`Test page URL: ${process.env.Url}`);
```

Comment:
Remove the `console.log` statement and use the strict URL comparison for all tests.

```js
import { pathToFileURL } from 'url';
import { resolve } from 'path';

const getFullFilePath = path => {
    const resolvedPath = resolve(__dirname, path);

    return pathToFileURL(resolvedPath).href;
};
...
test('Navigate to URL from an environment variable', async t => {
      await t.expect(getDocumentUri()).eql(getFullFilePath('./src/env.html'));
});

```

Reply:
Isnt that more logical:

`test('Navigate to URL from an environment variable', async t => {
      await t.expect(getDocumentUri()).eql(getFullFilePath(**process.env.url**));
});`